test/test_module_init.py

```diff
@@ -174,6 +174,9 @@ def build_constructor_arg_db():
         torch.nn.qat.Linear: ((5, 2), {
             'qconfig': torch.ao.quantization.default_qconfig,
         }),
+        torch.nn.qat.EmbeddingBag: ((10, 12), {
+            'qconfig': torch.ao.quantization.default_qconfig
```

Comment:
For EmbeddingBag don't we require the `float_qparams_qconfig`?

Reply:
Correct, coupled with checking for this in QAT EmbeddingBag's `__init__`, I've added a float qparam observer here for testing.  

Worth noting that many FakeQuant variants are not supported as qconfigs in this test (note `default_qconfig` for other QAT modules above as well).  This seems to be because default behavior for FakeQuantize scale is to use `torch.float32` instead of setting from `dtype` as is expected of Observers in this test.